pkg/columns/columninfo.go

```diff
@@ -49,6 +49,8 @@ type Attributes struct {
 	Name string `yaml:"name"`
 	// Name of the columns without inherited prefixes
 	RawName string `yaml:"raw_name"`
+	// Alias is an alternative shorter name to be used for header of the column; if not set, the Name will be used
+	Alias string `yaml:"alias"`
```

Comment:
Do you have to update the documentation of all gadgets if the column output is changed?

Reply:
Good question. I updated the documentation, but it's possible that it's out of sync. I think we'll need some automation if we want to keep it always updated, otherwise it'll be a very time consuming task.